libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -126,7 +126,8 @@ const static uint32_t   default_abi_serializer_max_time_us = 15*1000; ///< defau
  *  The number of sequential blocks produced by a single producer
  */
 const static int producer_repetitions = 12;
-const static int max_producers = 125;
+const static int max_producers = 125; // pre-savanna producer (proposer) limit
+const static int max_proposers = 64*1024; // savanna proposer (producer) limit
```

Comment:
If we expect them to always be the same, should we use one variable (for ex `max_proposers_finalizers`?)

Reply:
I don't think we do expect them to be the same. We have reason to believe we may want to reduce the max finalizers, where the max proposers seems like could remain at 64K.